lib/coverage/enabled.ml

```diff
@@ -0,0 +1,117 @@
+(** This module sets up the env variable for bisect_ppx which describes
+ *  where log files are written.
+ *)
+
+module D=Debug.Make(struct let name="coverage" end)
+let prefix = "org.xen.xapi.coverage"
+
+module Bisect = struct
+  let dump jobid =
+    let bisect_prefix = Unix.getenv "BISECT_FILE" in
```

Comment:
I think we should define the constant `"BISECT_FILE"` at the start of the Bisect module, then refer to it from here and from the `init_env` command.